root/static/scripts/tests/GuessCase.js

```diff
@@ -209,7 +209,7 @@ test('Work', function (t) {
     },
     {
       input: 'acte 1, no. 7: chœur: «voyons brigadier»',
-      expected: 'Acte 1, no. 7 : Chœur : « voyons brigadier »',
+      expected: 'Acte 1, no. 7 : Chœur : « Voyons brigadier »',
```

Comment:
This broke with this change, but I'm fairly sure a classical title like this should be capitalized to begin with and the previous test was bad.